redash/serializers/__init__.py

```diff
@@ -338,3 +341,68 @@ def serialize_job(job):
             "query_result_id": query_result_id,
         }
     }
+
+
+def get_location(ip):
```

Comment:
Maybe these functions can be collapsed into methods of `EventSerializer`, since they won't be used anywhere else.